BuildPropertiesPlugin/sample/app/build.gradle

```diff
@@ -1,15 +1,11 @@
 buildscript {
-  repositories {
-    maven { url rootProject.file('../.gradle/repo') }
-  }
   dependencies {
     classpath 'com.android.tools.build:gradle:2.1.2'
-    classpath "${pluginGroup}:${pluginId}:${pluginVersion}"
   }
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'build-properties'
+apply plugin: com.novoda.buildproperties.BuildPropertiesPlugin
```

Comment:
I see. The fully qualified name as a plug-in name is also recommended. Like 'com.android.application'  That's why I thought you changed it. 
:+1: